R/build_stan_data.R

```diff
@@ -67,40 +67,63 @@ sf_none <- function() {
 #' be estimated when sampling.
```

Comment:
corresponds

Reply:
I think we should say in addition something like: "The serofoi approach to fitting serological data currently supposes that FOI is piecewise-constant across either groups of years or ages, and this function creates a Data Frame that communicates this grouping to the Stan model."